src/components/DropdownButton/dropdownButton.stories.mdx

```diff
@@ -6,9 +6,9 @@ import {
   optionsKnob as options
 } from "@storybook/addon-knobs";
 import { ImportPath, State } from 'components/util/storybook';
-import DropdownButton from "./DropdownButton";
+import DropdownButton from ".";
```

Comment:
One thing to note: I tried using the `./` convention instead of just the dot but eslint (not sure in my code editor, or the project) got mad at me and said it's redundant and I should use this.